src/measurement/store.ts

```diff
@@ -54,7 +61,27 @@ export class MeasurementStore {
 		this.offloader = new MeasurementStoreOffloader(measurementStoreClient, this);
 	}
 
-	async getMeasurementString (id: string): Promise<string> {
+	async getMeasurementString (id: string): Promise<string | null> {
+		try {
+			const { minutesSinceEpoch, userTier } = parseMeasurementId(id);
```

Comment:
It can make sense to wrap only `const { minutesSinceEpoch, userTier } = parseMeasurementId(id);` in `try.. catch` to make expecting of error there explicit and remove nesting of two `try.. catch`es.